modules/contracts/src.ts/tests/integration/ethService.spec.ts

```diff
@@ -162,4 +159,8 @@ describe("EthereumChainService", function () {
     const res = await bobChainService.sendDefundTransferTx(transferState);
     expect(res.getValue()).to.be.ok;
   });
+
+  describe("revitalizeTxs", () => {
+    it("should start monitoring active txs", async () => {});
+  });
```

Comment:
What is this? Can we skip it at least? Comments would also be good

Reply:
Whoa that should not be there.